### PR TITLE
Shut it rubocop

### PR DESCRIPTION
**Pull request by jorge0136**
We are all tired of these warnings. 
image

One other small tweak to the `readme.md` as well.

### CODE REVIEW

**Review by optimumtk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ inherit_from:
 
 ## Updating This Gem
 
-After making changes to the rules in the [default.yml](default.yml) file, be sure that the version
+After making changes to the rules in the [optimum_energy_rubocop.yml.yml](optimum_energy_rubocop.yml.yml) file, be sure that the version
```
Comment (LandonSchropp) [inline README.md]: I think this line is a bit long now. Can you rewrap it and the ones below at 100 characters?

**Review by LandonSchropp — CHANGES_REQUESTED** [review]
Nice

File: README.md (hunk 1 above)
Comment (jorge0136) [reply README.md]: Fixed 👍

**Review by LandonSchropp — APPROVED** [review]

**Comment by jorge0136** [comment]
I bumped the rubocop version. I don't know what I am doing so I am asking for a re-review. Thanks :)

**Review by LandonSchropp — APPROVED** [review]

**Review by optijon — APPROVED** [review]

**Review by optimumtk — APPROVED** [review]

**Pull request merged**